sycl/test-e2e/Basic/multi_ptr_null_relational_operators.cpp

```diff
@@ -0,0 +1,55 @@
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
+#include <sycl/sycl.hpp>
+
+template <typename multi_ptr_t> void nullptrRelationalOperatorTest() {
+  constexpr int OUTPUT_SIZE = 8;
+  bool output[OUTPUT_SIZE];
+  try {
+    sycl::queue queue;
+    sycl::buffer<bool, 1> buf(output, sycl::range<1>(OUTPUT_SIZE));
```

Comment:
nit, my preference would be

```
  sycl::buffer<bool, 1> buf(N);
  /* do work */
  for (bool res : sycl::host_accessor{buf})
    assert(res);
```

until we will have ranges and `all_of` on a range.

Reply:
thanks @aelovikov-intel, done.